website-backend/docker/env.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Please only source this file, don't execute it directly.
+
+AWS_ACCOUNT_ID="012345678901"
+AWS_REGION="eu-central-1"
+ORG_NAME="rust-at-night"
+IMAGE_NAME="website-backend"
```

Comment:
If we want to add new sites this name may cause little confusion. Can we change it as homesite-backend or similar?

Reply:
Hmm.. actually it seems that you're right here. 
I'll be updating it.